src/main.py

```diff
@@ -24,3 +25,7 @@ async def readyz() -> dict:
 async def healthz() -> dict:
     """The endpoint for the Health Probe."""
     return {"healthy": "true"}
+
+
+if __name__ == "__main__":
+    uvicorn.run(app, host="localhost", port=8000)
```

Comment:
do you need this to run the app with `python src/main.py`?

Reply:
yes, if I want to debug it, I do need this, as the referenced docs of FastAPI shows.

Reply:
Somehow `fastapi` cli is the recommended way in the docs https://fastapi.tiangolo.com/#standard-dependencies. 
Can you share the doc that you read? I really used the way and there were some issue for me.

Reply:
https://fastapi.tiangolo.com/tutorial/debugging/